src/pages/index.js

```diff
@@ -29,51 +43,124 @@ const renderCard = (item) => {
 };
 
 //create objects
+//api
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "7ff811c0-9f09-42c2-975e-cca434c12992",
+    "content-type": "application/json",
+    "Access-Control-Allow-Origin": "*",
+  },
+});
+
 //profile section
-const userInformation = new UserInfo(".profile");
+const userInfo = new UserInfo(".profile");
+
+const profileSection = new Section(
+  {
+    renderer: () => {
+      api.getUserInformation().then(({ name, avatar, about, _id }) => {
+        userInfo.setAllInfo({
+          name: name,
+          avatar: avatar,
+          about: about,
+          _id: _id,
+        });
+      });
+    },
+  },
+  ".profile"
+);
 
 //cards section
 const cardList = new Section(
   {
-    data: initCards,
-    renderer: renderCard,
+    renderer: () => {
+      api.getInitialCards().then((items) => {
+        items.forEach((item) => {
+          renderCard(item);
+        });
+      });
+    },
   },
   ".page__cards"
 );
 
 //edit form
-const editFormValidator = new FormValidation("profile-edit-form", formData);
+const editFormValidator = new FormValidation("edit", formData);
```

Comment:
**Could be improved**

You can **universally** create instances of validators for all forms in the project storing them inside one object `formValidators`.  And then you can take any validator using attribute `name` of the form where you need to disable/enable the submit button or remove errors.

```js
const formValidators = {}

// enable validation
const enableValidation = (config) => {
  const formList = Array.from(document.querySelectorAll(config.formSelector))
  formList.forEach((formElement) => {
    const validator = new FormValidator(config, formElement)
    // here you get the name of the form
    const formName = formElement.getAttribute('name')

   // here you store a validator by the `name` of the form
    formValidators[formName] = validator;
   validator.enableValidation();
  });
};

enableValidation(config);

```

And now you can use them for disabling buttons or clearing errors:

```js

formValidators[ profileForm.getAttribute('name') ].resetValidation()

// or you can use a string – the name of the form (you know it from `index.html`)

formValidators['profile-form'].resetValidation()

```